pages/sdk/relay-kit/reference/safe-4337-pack.mdx

```diff
@@ -89,7 +89,7 @@ PaymasterOptions = {
 ```
 
 - **`provider`** : The EIP-1193 compatible provider or RPC URL of the selected chain.
-- **`signer`** : The signer private address if the `provider` doesn't resolve to a signer account. If the `provider` resolves to multiple signer addresses, the `signer` property can be used to specify which account to connect, otherwise the first address returned will be used.
+- **`signer`** : A passkey or the signer private address if the `provider` doesn't resolve to a signer account. If the `provider` resolves to multiple signer addresses, the `signer` property can be used to specify which account to connect, otherwise the first address returned will be used.
```

Comment:
```suggestion
- **`signer`** : A passkey or the signer private key if the `provider` doesn't resolve to a signer account. If the `provider` resolves to multiple signer addresses, the `signer` property can be used to specify which account to connect, otherwise the first address returned will be used.
```